### ISSUE TITLE
Add script to special case "value" in formField tokens

### ISSUE THREAD

**Issue by taysea** [grommet/hpe-design-system#4356]
Ideally the formField tokens can read:
- `formField.value.textColor`
- `formField.placeholder.textColor`

rather than the current `valueText`, `placeholderText` which was done because "value" is a special word for style-dictionary.

Deliverable:
- [x] Modify the build script for style-dictionary such that "value" in the formfield context is identified as a special case and resolved properly.

**Comment by taysea** [comment]
style-dictionary v4 is now released and aligns closer to the design token spec with use of `$value` for the reserved term.

In a small demo project I tested that use of `value` wouldn't cause issues.

![Image](https://github.com/user-attachments/assets/208dbc2a-362b-4c1c-b467-c549664d7383)

![Image](https://github.com/user-attachments/assets/c36ff9a5-b2e6-451b-a9b1-fd973dd58bcd)

I think the best path forward is to upgrade to style-dictionary v4.
